utils/compare_dependency_constraints.py

```diff
@@ -93,8 +110,18 @@ def compare_constraints(images_contained_in_native: list[str]):
             )
         )
 
-        for discrepancy in discrepancies:
-            print(str(discrepancy))
+    for discrepancy in discrepancies:
+        file_pyproject_path = DOCKER_FOLDER / f"{discrepancy.image}/pyproject.toml"
+        if file_pyproject_path.exists():
+            file_path = file_pyproject_path
+        else:
+            file_path = DOCKER_FOLDER / f"{discrepancy.image}/Pipfile"
```

Comment:
We should know the exact file path when creating the discrepancy object, let's add it at that step rather than search for it here